packages/state/source/wallets/wallet-repository-clone.ts

```diff
@@ -21,6 +19,11 @@ export class WalletRepositoryClone extends WalletRepository implements Contracts
 		}
 	}
 
+	public setOriginalWalletRepository(originalWalletRepository: WalletRepository): WalletRepositoryClone {
```

Comment:
Maybe `configure` is more fitting, because that's how we usually call it in other places (e.g. `validator.ts` )